Stratis.Bitcoin.Features.Wallet/WalletFeature.cs

```diff
@@ -74,6 +74,24 @@ public void AddNodeStats(StringBuilder benchLogs)
         }
 
         /// <inheritdoc />
+        public void AddFeatureStats(StringBuilder benchLog)
+        {
+            var walletNames = this.walletManager.GetWalletsNames();
+
+            if (walletNames.Any())
+            {
+                benchLog.AppendLine();
+                benchLog.AppendLine("======Wallets======");
+
+                foreach (var walletName in walletNames)
```

Comment:
you are still in war with var, here and down there and up there, everywhere ;)

Reply:
This is just a string so I would say var is fine.

Reply:
How do you know it is a string if you see `var`?